pkg/auth/tokens/manager.go

```diff
@@ -630,7 +630,8 @@ var uaBackoff = wait.Backoff{
 
 func (m *Manager) NewLoginToken(userID string, userPrincipal v3.Principal, groupPrincipals []v3.Principal, providerToken string, ttl int64, description string) (v3.Token, string, error) {
 	provider := userPrincipal.Provider
-	if (provider == "github" || provider == "azuread" || provider == "googleoauth") && providerToken != "" {
+	// Providers that use oauth create need to create a secret for storing the access token.
```

Comment:
Was this typo or on purpose ? I don't think `oauth create` is a special thing
```suggestion
	// Providers that use oauth need to create a secret for storing the access token.
```